action/receipt.go

```diff
@@ -146,9 +192,18 @@ func (log *Log) ConvertFromLogPb(pbLog *iotextypes.Log) {
 	log.BlockHeight = pbLog.GetBlkHeight()
 	copy(log.ActionHash[:], pbLog.GetActHash())
 	log.Index = uint(pbLog.GetIndex())
+
+	if len(log.Topics) > 1 {
+		for _, l := range log.Topics[1:] {
+			if log.Topics[0] == l {
+				log.NotFixTopicCopyBug = true
+				break
+			}
+		}
+	}
```

Comment:
invalid statement, revert it

Reply:
i add this b/c i saw the newly added TestLogs() would fail with `log.NotFixTopicCopyBug = true`

so add this check, if all topics are same --> the log is earlier than this fix, so set this flag = true

it is not absolutely necessary, but makes the code more accurate